allennlp/data/dataset_readers/language_modeling.py

```diff
@@ -57,7 +59,7 @@ def read(self, file_path: str):
             tokenized_text = self._tokenizer.tokenize(all_text)
             num_tokens = self._tokens_per_instance
             tokenized_strings = []
-            for index in range(0, len(tokenized_text) - num_tokens, num_tokens):
+            for index in tqdm.tqdm(range(0, len(tokenized_text) - num_tokens, num_tokens)):
```

Comment:
Can you put a logging statement above all calls to tqdm like this, saying what's being iterated over?